automation/roles/pgbackrest/tasks/auto_conf.yml

```diff
@@ -10,11 +9,26 @@
         - { option: "log-path", value: "/var/log/pgbackrest" }
         - { option: "repo1-type", value: "s3" }
         - { option: "repo1-path", value: "{{ PGBACKREST_REPO_PATH | default('/pgbackrest') }}" }
-        - { option: "repo1-s3-key", value: "{{ PGBACKREST_S3_KEY | default(lookup('ansible.builtin.env', 'AWS_ACCESS_KEY_ID')) }}" }
-        - { option: "repo1-s3-key-secret", value: "{{ PGBACKREST_S3_KEY_SECRET | default(lookup('ansible.builtin.env', 'AWS_SECRET_ACCESS_KEY')) }}" }
-        - { option: "repo1-s3-bucket", value: "{{ PGBACKREST_S3_BUCKET | default(aws_s3_bucket_name | default(patroni_cluster_name + '-backup')) }}" }
-        - { option: "repo1-s3-endpoint", value: "{{ PGBACKREST_S3_ENDPOINT | default('s3.' + (aws_s3_bucket_region | default(server_location)) + '.amazonaws.com') }}" }
-        - { option: "repo1-s3-region", value: "{{ PGBACKREST_S3_REGION | default(aws_s3_bucket_region | default(server_location)) }}" }
+        - {
+            option: "repo1-s3-key",
+            value: "{{ PGBACKREST_S3_KEY | default(lookup('ansible.builtin.env', 'AWS_ACCESS_KEY_ID')) }}",
+          }
```

Comment:
It looks quite readable and compact in a single line. The lines are not long enough to justify such formatting.

This comment applies to the rest of the code lines.

Reply:
See my previous comments about `prettier`.